lib/availability-phrase-resources.js

```diff
@@ -12,7 +12,7 @@ var otherLocalisedStrings = {
     EveryDay: "Diario"
   },
   ja: {
-    EveryDay: "毎日の"
+    EveryDay: "月〜日（毎日）"
```

Comment:
I see that there is a space at the very end of the string; did you check w/ Miwa if this is alright?

// cc @arcaartem @wxenakis19